src/components/WebLogView/WebLogView.utils.ts

```diff
@@ -20,3 +22,12 @@ export function isGroupedProxyData(
 ): data is GroupedProxyData {
   return !Array.isArray(data)
 }
+
+export function getRequestType(data: ProxyData) {
+  if (isNonStaticAssetResponse(data)) {
+    return 'fetch'
```

Comment:
ChromeDevTools categorizes this a bit more. I was debating whether to call it "document", "dynamic" or "fetch". Any thoughts?

Reply:
Perhaps, we can make use of the content-type header to categorize different types of dynamic requests?